### DISCUSSION TITLE
Quarkus World Tour 2023 - Call for Roadies

### DISCUSSION THREAD

**Discussion by insectengine** [quarkusio/quarkus#23454] [Events]
The 2023 Quarkus World Tour is kicking off once again to provide a unique, hands-on Quarkus experience for Java developers across the globe. The goal of the tour is to introduce Quarkus to Java developers as well as take a deep dive into the technology.

Since the tour began, we have performed for crowds of developers from 25+ JUGs across the world with many more tour stops being added every day.

**Bands Make It “Rock”, Roadies Make It “Roll”**
A global world tour takes an entire community to ensure its success. We have a great group of roadies that have made the world tour a success so far but we need your help.

**Ways You Can Help**
Much like a concert, we want each tour stop to be a unique experience for the audience. Each performer is encouraged to demo or talk about what they are passionate about with Quarkus. You DO NOT have to be an expert or know everything there is about Quarkus.

The event organizer will work with you ahead of time to set the agenda and organize the demo setlist so you have time to prepare (see sample setlist below). We have a repository of content that you can choose from or customize so you don’t have to build something from scratch. Sign up for one location or as many as you feel comfortable supporting.

**_Performers are needed for the following roles:_**
- MC (kick things off, transition between demos, monitor chat and questions, etc.)
- Presenter (demo Quarkus feature/capability of your choosing)
- Organizer (help arrange logistics, agenda, JUG communication, etc.)

**Comment by xstefank** [comment]
Sign me up :). Can you point me to where can I officially volunteer for roadie?

**Reply by charger28** [reply]
Martin. You are in. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Comment by evrentan** [comment]
Hi @insectengine,
I am a volunteer to become a roadie for Quarkus World Tour 2022, especially for the stop in Turkey as I requested for our JUG, Turkey Java Community. Pls find the comment for request a tour stop in the below link;
https://github.com/quarkusio/quarkus/discussions/23453#discussioncomment-2181573

Thanks,
Evren

**Reply by charger28** [reply]
Evren. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Comment by cgruver** [comment]
I would love to participate.

**Reply by charger28** [reply]
Charro. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Comment by hugoloza** [comment]
i'm a roadie volunteer!  Quarkus + Rock .. what an incredible combination. i can help as mc or Presenter.

**Reply by charger28** [reply]
Hugo. You are in. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Comment by aruffie** [comment]
@insectengine  @cescoffier @gsmet up for me, I'm volunteer on Toulouse if I can help :+1:

**Reply by charger28** [reply]
Ruffie. You are in. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Reply by charger28** [reply]
@aruffie - can you send your contact information to worltour@quarkus.io and provide your contact information?

**Reply by aruffie** [reply]
@charger28  of course, but what do you mean about contact information ? Phone, address, other ?

**Reply by charger28** [reply]
@aruffie - just your email for now so that we can reach out to you about more information on being a Quarkus World Tour roadie. Thanks

**Comment by LightGuard** [comment]
I'm interested in helping again this year.

**Reply by charger28** [reply]
Jason - we would love to have you help again. We will follow up soon.

**Comment by maxandersen** [comment]
Greetings all - fyi @insectengine is out on PTO this week - he'll catch up with you next week.

**Reply by aruffie** [reply]
all rights, nothing isn't urgent

**Comment by viniciusfcf** [comment]
I can help with the tour here in Brazil in Portuguese

**Reply by charger28** [reply]
Vinicius. You are in. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Reply by tiago-marques** [reply]
@charger28 I would like to help too, if have any more seats for that ! Same tour: Brazil, languages: english, portuguese and spanish

**Reply by charger28** [reply]
@tiago-marques - thanks for volunteering. We will follow up soon.

**Comment by wjglerum** [comment]
Great idea, I can help as a presenter in The Netherlands

**Reply by charger28** [reply]
Willem. You are in. We will follow up soon with more details. We are working to schedule a number of tour stops then we will reach out to the roadies crew for signups.

**Reply by wjglerum** [reply]
Hi! Do you have any updates? I could also help around and assist in both The Netherlands and Belgium if you want.

**Comment by NRaoV** [comment]
Great Idea, I can help as a presenter in India-Bangalore & Chennai. Please sign me in. Thanks

**Reply by charger28** [reply]
Nageswara. Thanks for reaching out. We will add you to the Quarkus World Tour roadie list. We will be reaching out soon.

**Comment by loicmathieu** [comment]
Hi,
You can count me on for any tour in France or Belgium assuming it's still virtual.
I can help on tour in english also if the timezone is not far from Paris time.

**Reply by charger28** [reply]
Thanks Loic. You are on the list.

**Comment by charger28** [comment]
Roadies - thank you so much to all of you who have volunteered for the 2022 Quarkus World Tour. We apologize for the delay getting things rolling but we are currently working to solidify 15-20 tour stop dates. Once we have the dates, we will reach out to everyone and have you sign up for the tour stops that you would like to participate in. We anticipate sending emails by the end of this week. Thanks for your patience.

**Comment by gbengataylor** [comment]
If you still need volunteers in NA, I'll be happy to assist

**Comment by LightGuard** [comment]
Here's an update on what we have planned and where we need some help.

* May 17 7:00 PM UTC+2 Orleans, Orleans JUG France In-Person: We need a presenter
* May 21 TBD Turkey (TBD) Turkey Java Community In-Person: We need a presenter
* May 26 12:30 UTC+1 Manchester UK, Manchester JUG Virtual: We need a presenter
* May 31 Barcelona Spain, Barcelona JUG In-Person: All ready, Alex Soto will present, please attend
* June 9 TBD Bordeaux France, Bordeaux JUB In-Person: We need a presenter
* June 14 (TBD) Toulouse France, Toulouse JUG In-Person: We need a presenter
* June (TBD) Grenoble France, Alpes JUG In-Person: We need a presenter
* June (TBD) Nancy France, Lorraine JUG In-Person: We need a presenter
* June (TBD) Montreal Canada, Montreal JUG In-Person: We need a presenter
* July 27 Frankfurt Germany, JUGF In-Person: We need a presenter
* July 28 Chicago Illinois USA: All ready, please attend
* September 14 Prague Czechia, CZ JUG In-Person: All ready, Martin Stefanko and Martin Kouba will present, please attend

We have some slides and demos ready for anyone who is willing to present. Most of these presentations would prefer a native language presenter, but will accept English as a backup. Please let us know if you're able to help present on this year's Quarkus World Tour.

**Comment by erik-wramner** [comment]
I guess I'm late to the game here, but if you decide to add a stop in Sweden I can present. That also applies if this is repeated in Sweden next year.

**Comment by coiouhkc** [comment]
I'm preparing a Quarkus intro for an internal presentation, if you decide to make a stop in Germany, I could present (https://github.com/coiouhkc/short-and-sweet-quarkus-intro, interim name "10 things I like about Quarkus").

**Comment by bentito** [comment]
I'd love to help if you come to Bordeaux or Toulouse!

**Comment by andotorg** [comment]
i'm a roadie volunteer! Quarkus i can help as mc or Presenter.

**Comment by mariasalcedo** [comment]
I would love to help anywhere in Germany

**Comment by dvsingh9** [comment]
Am I late here? I can help in Delhi NCR(India) area and Eindhoven(Netherlands)
